qwik-graphql-tailwind/src/routes/[user]/index.tsx

```diff
@@ -66,15 +72,13 @@ export default component$(() => {
       <div className="mx-auto max-w-screen-2xl py-8 px-4">
         <div className="grid grid-cols-12 gap-8">
           <div className="pt-8 relative z-20 col-span-12 md:-top-20 md:col-span-4 xl:col-span-3">
-            {store.user ? <UserProfileCard {...store.user} /> : 'Loading...'}
+            {store.user ? <UserProfileCard {...store.user} /> : null}
```

Comment:
`store.user && <UserProfileCard {...store.user} />` instead

Reply:
This was done based on a recent finding; you can read it up here https://medium.com/geekculture/stop-using-for-conditional-rendering-in-react-a0f7b96200f8